docs/userguides/config.md

```diff
@@ -37,6 +37,55 @@ plugin:
 
 This helps keep your secrets out of Ape!
 
+If a configuration is left unset (i.e., not included in the
+`ape-config.(yaml|json|toml)` file, Ape will optionally inspect the
+environment variables as a fallback, following the pattern
+"<PREFIX>_SETTING", where different types of configurations have
+different prefixes.
+
+For example, the following config:
+
+```yaml
+contracts_folder: src/qwe
+test:
+  number_of_accounts: 3
+  show_internal: True
+compile:
+  exclude:
+    - "one"
+    - "two"
+    - "three"
+  include_dependencies: true
+```
+
+could be entirely defined with environment variables as follows:
+
+```shell
+APE_CONTRACTS_FOLDER=src/contracts
+APE_TEST_NUMBER_OF_ACCOUNTS=3
+APE_TEST_SHOW_INTERNAL=true
+APE_COMPILE_EXCLUDE='["one", "two", "three"]'
+APE_COMPILE_INCLUDE_DEPENDENCIES=true
+```
+
+Here is the complete list of supported prefixes:
+
+| Module       | Prefix       |
```

Comment:
(i can fix these with my nitty commits), but just to clarify, these are not actually modules but are packages shipped along with ape! like `ape.ethereum` would be a module but `ape_ethereum` is a package.  that is the nature of ape's plugin architecture, we ship plugins with ape so it works out of the box.